crates/ruff_formatter/src/macros.rs

```diff
@@ -320,17 +320,17 @@ macro_rules! format {
 /// the content up to the first non-soft line break without exceeding the configured print width.
 /// This definition differs from groups as that non-soft line breaks make group expand.
 ///
-/// [crate::BestFitting] acts as a "break" boundary, meaning that it is considered to fit
+/// [crate::FormatBestFitting] acts as a "break" boundary, meaning that it is considered to fit
 ///
 ///
 /// [`Flat`]: crate::format_element::PrintMode::Flat
 /// [`Expanded`]: crate::format_element::PrintMode::Expanded
-/// [`MostExpanded`]: crate::format_element::BestFitting::most_expanded
+/// [`MostExpanded`]: crate::format_element::FormatBestFitting::most_expanded
```

Comment:
Failure in documentation generation because this should be `BestFitting`

```
error: unresolved link to `crate::format_element::FormatBestFitting::most_expanded`
   --> crates/ruff_formatter/src/macros.rs:328:23
    |
328 | /// [`MostExpanded`]: crate::format_element::FormatBestFitting::most_expanded
    |                       ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ no item named `FormatBestFitting` in module `format_element`
    |
```

Do I need to expose `BestFitting` at https://github.com/charliermarsh/ruff/blob/90d47a5980a4b2fd47dabedfe5f2acd7b4081fcf/crates/ruff_formatter/src/lib.rs#L51 and use it here instead? I'm surprised tests passed if this is incorrect as-is.

Reply:
`from_arguments_unchecked` is on `FormatBestFitting` so perhaps the docs link is what is wrong

Reply:
https://github.com/charliermarsh/ruff/pull/4841/commits/28cd3afadabeb3e037da0c22c46bb4b213ee23b9